README.md

```diff
@@ -637,7 +662,12 @@ Detailed list can be found [here](http://bit.ly/3GnJwgZ)
 - [DarkNet-53](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/darknet53.py)
 - [SSD (Single Shot Detector)](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/ssd.py) 
 - [YOLOX](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/yolox.py)
-  
+
+
+### Pose Estimation
+
+- [DEKR-W32-NO-DC](super_gradients/recipes/coco2017_pose_dekr_w32_no_dc.yaml)
```

Comment:
Link not working

Reply:
Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2